.github/workflows/continuous-integration.yml

```diff
@@ -83,7 +86,10 @@ jobs:
     needs: build
     services:
       db:
-        image: postgres:11.2
+        image: postgres:14.2
+        env:
+          POSTGRES_USER: postgres
```

Comment:
Why are these env variables needed if they are also explicitly defined below?

Reply:
This is not a `docker-compose.yml` file. The environment variables here are used to set up the container, but they still need to be supplied to the command below in order for the application to be able to use them.